iterfzf/__init__.py

```diff
@@ -43,8 +43,15 @@ def iterfzf(
         cmd.append('+i' if case_sensitive else '-i')
     if exact:
         cmd.append('--exact')
-    if multi:
-        cmd.append('--multi')
+    if cycle:
+        cmd.append('--cycle')
+    if multi is not None:
+        if type(multi) == bool and multi:
```

Comment:
As built-in types also can be subclassed, we'd better to check types using `isinstance()`.

```suggestion
        if isinstance(multi, bool) and multi:
```